### ISSUE TITLE
Node HTTPRequest only use `url` from property while having an input `url`

### ISSUE THREAD

**Issue by liusida** [jagenjo/litegraph.js#490]
![image](https://github.com/jagenjo/litegraph.js/assets/14249458/f534c2d6-d8c6-48d8-b8e1-79a4cf021b3b)

This is not working unless I set the `url` in HTTPRequestNode's property. 

Maybe it should use the input `url` instead of the property `url`.

https://github.com/jagenjo/litegraph.js/blob/0555a2f2a3df5d4657593c6d45eb192359888195/src/nodes/network.js#L384

Thanks~

**Comment by atlasan** [comment]
You are correct, that's a bug, that node was added and not correctly finished up.
ATM the good way to use both property and input would be to use _getInputOrProperty_ instead, a maybe better way would be to have eventually unified toggable property/widget/input
